web/src/config.rs

```diff
@@ -1,19 +1,26 @@
 use http::Uri;
 
 const NAME_SERVICE_URI: &str = "https://factory.c4dt.org/incubator/at2/demo/ns";
+const NETWORK_URI: &str = "https://factory.c4dt.org/incubator/at2/demo/leader";
 
 pub struct Config {
     name_service: Uri,
+    network: Uri,
 }
 
 impl Config {
     pub fn parse() -> Result<Self, http::uri::InvalidUri> {
```

Comment:
Do I understand correctly that this is a stub and should later be replaced by a real config-file? If yes, please add a `TODO` comment and say so.

Reply:
Right, it's temporary, adding a `TODO` line.